changelogs/fragments/7984-new-list-filters-for-union-intersection-difference-added.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - New list filters ``community.general.lists_union``, ``community.general.lists_intersect``, and ``community.general.lists_difference`` added. These filters preserve the items order and can be used instead of the builtin one which start to behave differently with Ansible Core >= 2.16.
```

Comment:
Please remove this. New filters are announced automatically based on their `version_added` value.